basic-solutions/xdp_stats.c

```diff
@@ -289,6 +291,7 @@ int main(int argc, char **argv)
 		err = stats_poll(pin_dir, stats_map_fd, info.id, info.type, interval);
 		if (err < 0)
 			return err;
+		close(stats_map_fd);
```

Comment:
This still leaks the fd in the error paths

Reply:
Yes, of course

Reply:
Great, thanks!